### PR TITLE
Update script with curly braces in the script 

### PR DESCRIPTION
**Pull request by radoondas**
This is simple update which adds 'curly-braces' 
Extending: elastic/beats-dashboards#83

### CODE REVIEW

File: load.sh [hunk 1]
```diff
@@ -74,7 +74,7 @@ case $1 in
 
      *)
         echo "Error: Unknown option $2"
-        print_usage
+        print_usageUSER
```
Comment (andrewkroh) [inline load.sh]: What is this change?

**Comment by andrewkroh** [comment]
Except for the one comment, LGTM.

It seems like #83 can be closed because the `s/name/NAME/g` change is implemented here.

File: load.sh (hunk 1 above)
Comment (radoondas) [inline load.sh]: Ouch, this is wrong and :( not planned typo, and why we have to check the code. I have to delete 'USER' part of the string.

**Comment by ruflin** [comment]
LGTM except for the part discovered by @andrewkroh . @radoondas Let us know when you pushed the change as Github does not send notifications for new commits.

**Comment by radoondas** [comment]
@ruflin I have just pushed fix to github. Could someone please double check?

**Comment by ruflin** [comment]
@radoondas LGTM. Merged.

**Pull request merged**